iota/adapter/__init__.py

```diff
@@ -52,12 +41,8 @@
 """
 
 # Load SplitResult for IDE type hinting and autocompletion.
-if PY2:
-    # noinspection PyCompatibility,PyUnresolvedReferences
-    from urlparse import SplitResult
-else:
-    # noinspection PyCompatibility,PyUnresolvedReferences
-    from urllib.parse import SplitResult
+# noinspection PyCompatibility,PyUnresolvedReferences
+from urllib.parse import SplitResult, urlsplit
```

Comment:
❔ I think the `#noinspection` line can also be removed (it is there to suppress IDE warnings about PY2-/PY3-incompatible imports, which no longer applies).

Reply:
Okay!